### PR TITLE
LPS-150313 Conslidate Publications resource action configs into the service module

### PR DESCRIPTION
**Pull request by stsquared99**

### CODE REVIEW

**Comment by stsquared99** [comment]
@tinatian Does this look right? Pending more comprehensive test results

**Comment by stsquared99** [comment]
ci:test

**Comment by stsquared99** [comment]
Will need to rebase and resend

File: modules/apps/change-tracking/change-tracking-service/src/main/resources/resource-actions/default.xml [hunk 1]
```diff
@@ -2,6 +2,25 @@
 <!DOCTYPE resource-action-mapping PUBLIC "-//Liferay//DTD Resource Action Mapping 7.4.0//EN" "http://www.liferay.com/dtd/liferay-resource-action-mapping_7_4_0.dtd">
 
 <resource-action-mapping>
+	<portlet-resource>
```
Comment (tinatian) [inline modules/apps/change-tracking/change-tracking-service/src/main/resources/resource-actions/default.xml:5]: @stsquared99 <portlet-resource> should be defined in ***-web as it is where Portlet class stays. Why you need to move it to ***-service module?

**Pull request closed without merging**